src/main/java/com/xceptance/xlt/engine/resultbrowser/DomUtils.java

```diff
@@ -47,6 +47,11 @@
  */
 final class DomUtils
 {
+    /**
+     * Class logger.
+     */
+    private static final Logger LOGGER = Logger.getLogger(DomUtils.class);
```

Comment:
When creating class-level loggers elsewhere, we usually use the commons-logging approach:

```private static final Log LOG = LogFactory.getLog(DomUtils.class);```

I am aware that the previously used runtime logger is also a plain log4j logger. However, this has been wrong all the time, but could not be changed because it's API.

WDYT about changing this to commons-logging style?

Reply:
Changed to commons-logging.